.travis.yml

```diff
@@ -1,7 +1,10 @@
-sudo: false
+sudo: required
 language: rust
 rust:
   - nightly-2017-10-16
+before_install:
+  - sudo apt-get update -q || true
```

Comment:
It's a dirty workaround. 😅

Reply:
Hopefully we won't have to keep this forever :)